services/_shared/llm_client.py

```
@@ -1464,6 +1518,120 @@ def review_diff(
+            config = _saas_overload_fallback_config(backend)
+            start_ns = time.monotonic_ns()
+            with _llmobs_llm(
+                model_name=config.model, model_provider=backend.value, name=_LLMOBS_NAME,
+            ) as span:
+                try:
+                    resp = _call_backend(config, messages)
+                except _BackendConfigError as e:
+                    log.error(
+                        "llm_backend_misconfigured",
+                        extra={"backend": backend.value, "detail": str(e)},
+                    )
```

Ruff (0.15.21)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)